### DISCUSSION TITLE
leaflet maps not responsive on modal bootstrap

### DISCUSSION THREAD

**Discussion by akbar-ardiansyah** [community/community#18343] [Discussions] [answered]
i want to show leaflet map when modal appears
I have tried various ways, but it still doesn't work.
I think this is a conflict with the bootstrap class, please solve it, friends

`$('#maps').ready(function () {
        var map = L.map('maps').setView([res.latitude, res.longtitude], 15);
        L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw', {
          maxZoom: 18,
          id: 'mapbox/streets-v11',
          tileSize: 512,
          zoomOffset: -1,
        }).addTo(map);

        var popup = L.popup()
          .setLatLng([res.latitude, res.longtitude])
          .setContent(res.nama_objek_wisata)
          .openOn(map);

        var circle = L.circle([res.latitude, res.longtitude], {
          color: 'cornflowerblue',
          fillColor: '#6494ed8e',
          fillOpacity: 0.5,
          radius: res.luas

        }).addTo(map);
      });`
<img width="1680" alt="Screen Shot 2022-06-11 at 15 52 55" src="https://user-images.githubusercontent.com/73751746/173180990-e8928ae5-c412-45fc-8e7d-d217603f7eb5.png">

**Comment by artabed** [comment, marked as answer]
map.invalidatesize(true)
